src/main/pg/alt_hold.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#include "pg/pg.h"
+
+typedef struct altholdConfig_s {
+    uint8_t alt_hold_pid_p;
+    uint8_t alt_hold_pid_i;
+    uint8_t alt_hold_pid_d;
+    uint8_t alt_hold_target_adjust_rate;
```

Comment:
Max is 2.5m/s. Is that enough?

Reply:
I think so.  For landing purposes, 1m/s as the maximum seems ideal.  For low-altitude stabilised hovers it is enough also.  One downside of having significant throttle control is that if throttle is accidentally bumped in altitude hold, the quad may then unexpectedly crash or climb excessively.  Hence I think a relatively slow adjustment rate makes sense. If someone wants to move quickly up and down, they can exit altitude hold, throttle up or down, and re-establish altitude wherever they intend to stay.